main.ts

```diff
@@ -18,34 +18,40 @@ const flags = parse(Deno.args, {
   default: {
     "browser-exec-path": "/usr/bin/chromium",
     headless: true,
-    ignore: ".ignore",
+    ignore: ".trunignore",
```

Comment:
Should get rid of this default?

Reply:
i think its fine. just like how npm has a `.npmginore` or how git has a `.gitignore` we can have a `.trunignore`. And having a .`npmignore` or `.gitingore` isn't required for npm or git to work.